Packs/RsaNetWitnessPacketsAndLogs/Integrations/RSANetWitnessPacketsAndLogs/README.md

```diff
@@ -1039,11 +1067,14 @@
 <td style="width: 97px;">concentratorPort</td>
 <td style="width: 61px;">uint32 </td>
 <td style="width: 455px;">
-<p>Port to use with the concentratorIP parameter</p>
-<ul>
-<li>SSL: 56105</li>
-<li>Non-SSL: 50105</li>
-</ul>
+<p>Port to use with concentratorIP 50105 (SSL/NON-SSL) for concentrator, 50103 (SSL/NON-SSL) for broker.</p>
+</td>
+</tr>
+<tr>
+<td style="width: 97px;">useSSL</td>
+<td style="width: 61px;">boolean</td>
+<td style="width: 508px;">
+<p>Send request in HTTPS instead of HTTP. By default the parameter is set to True but if your appliance is not configure to use SSL you can change it here.</p>
```

Comment:
```suggestion
<p>Send request in HTTPS instead of HTTP. By default the parameter is set to True but if your appliance is not configured to use SSL you can change it here.</p>
```